src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Blocks/tabs.html.twig

```diff
@@ -34,7 +34,7 @@
     <li id="tab_step2" class="nav-item"><a href="#step2" role="tab" data-toggle="tab" class="nav-link">{{ 'Pricing'|trans({}, 'Admin.Catalog.Feature') }}</a></li>
     <li id="tab_step5" class="nav-item"><a href="#step5" role="tab" data-toggle="tab" class="nav-link">{{ 'SEO'|trans({}, 'Admin.Catalog.Feature') }}</a></li>
     <li id="tab_step6" class="nav-item"><a href="#step6" role="tab" data-toggle="tab" class="nav-link">{{ 'Options'|trans({}, 'Admin.Global') }}</a></li>
-    {% if hookcount('displayAdminProductsExtra') > 0 %}
+    {% if hooksarraycontent(hooks) is not empty %}
```

Comment:
What if this template was used before, `hooks` is not defined in this one :thinking: 
Be sure the variable is properly passed to this template